go.mod

```diff
@@ -42,20 +43,20 @@ require (
 	github.com/rancher-sandbox/ele-testhelpers v0.0.0-20221213084338-a8ffdd2b87e3
 	github.com/rancher-sandbox/go-tpm v0.0.0-20220823075603-d273b298fcda
 	github.com/rancher/fleet/pkg/apis v0.0.0-20220318160658-4dc66c946ca2
-	github.com/rancher/lasso v0.0.0-20210709145333-6c6cd7fd6607
+	github.com/rancher/lasso v0.0.0-20220628160937-749b3397db38
 	github.com/rancher/rancher/pkg/apis v0.0.0-20211013185633-a636bda2a00e
 	github.com/rancher/steve v0.0.0-20220503004032-53511a06ff37
-	github.com/rancher/system-agent v0.2.8
+	github.com/rancher/system-agent v0.2.14-0.20230106171935-24c523a4408a
 	github.com/rancher/system-upgrade-controller/pkg/apis v0.0.0-20220321230925-79da9f058209
-	github.com/rancher/wrangler v0.8.11-0.20220217210408-3ecd23dfea3b
```

Comment:
There are still some usages of wrangler code here.. I can put up an issue to migrate away those few cases!

Reply:
The most critical wrangler usage in our code is for the steve aggregation client: that's the way we get the URL to serve our operator APIs from Rancher URL. We can try to migrate that, but it is not urgent IMO (moreover I'm not sure if it would work flawlessly with Rancher using "just" the vanilla kubernetes API extension, so may take some time to migrate).